docs/stencil-docs/reference-docs/handlebars-helpers-reference.mdx

```diff
@@ -286,6 +289,60 @@ The following script is an example of Early Hints usage in the Cornerstone theme
 
 You can view more Early Hints usage in Cornerstone's `templates/layout/base.html` file by visiting the [Cornerstone Repository](https://github.com/bigcommerce/cornerstone/commit/f70fcd502343503969c2e475b777a85347137542#).
 
+### EarlyHint
+
+```handlebars showLineNumbers
+{{ earlyHint '{path}' '{REL}' [as='{AS}'] [crossorigin='{CORS}']}}
+```
+
+Retrieves the provided URL and produces an early hint to communicate important web page resources. The server indicates which site assets (such as CSS files, JS scripts, and fonts) a browser needs to render a page fully. When used in conjunction with the `{{cdn}}` helper, load times can improve by 30%.
+
+#### Parameters
+
+- `path` (String): The provided path used when determining web page resources.
+- `REL` (String): Valid values are `preload`, `preconnect`, `prerender`,  and `dns-prefetch`.
+- `AS` (String): Optional. Valid values are `script`, `style`, `font`.
+- `CORS` (String): Optional. Valid values are `no` (default), `anonymous`, and `use-credentials`.
+
+
+#### Examples
+
+```handlebars showLineNumbers
+{{earlyHint 'https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js' 'preload' as='script' crossorigin='anonymous'}}
+```
+This example produces an early hint in a `Link` header that looks like:
+
+```handlebars showLineNumbers
+<https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js>; rel=preload; as=script; crossorigin=anonymous
+```
+
+
+The next example includes additional helpers. `getFontsCollection` and `stylesheet` helpers produce early hints by default. 
+
+
+```handlebars showLineNumbers

```

Comment:
This example shows the cdn helper, but not the earlyHint helper.

Reply:
That's correct. The case I tried to make here is to signal the helpers that produce early hints apart from `earlyHint`. I don't know if it would be better to signal those details in each helper section.

Reply:
i think we just need to say that above, cc @bc-traciporter